Frontend/android-native/mobility-common/src/main/java/in/juspay/mobility/common/MediaPlayer.java

```diff
@@ -119,6 +120,10 @@ public void uploadFile(int imageAspectHeight, int imageAspectWidth, boolean show
         }
     }
 
+    public void takePhoto(){
```

Comment:
where is the function being used?

Reply:
If it is coming from jBridge can we make this more configurable?

Reply:
this call uploadfile with canChooseFromFile param : which sets whether user can upload the media from file also or directly from camera

Reply:
we can directly call this uploadFile from Jbridge with canChooseFromFile param.